addons/account/models/account_journal.py

```diff
@@ -1094,3 +1114,52 @@ def _process_reference_for_sale_order(self, order_reference):
         '''
         self.ensure_one()
         return order_reference
+
+    # -------------------------------------------------------------------------
+    # E-Invoice Related Methods
+    # -------------------------------------------------------------------------
+
+    @api.model
+    def _postprocess_received_einvoices(self, moves):
+        journal = moves.mapped('journal_id').ensure_one()
```

Comment:
```suggestion
    def _postprocess_received_einvoices(self):
        self.ensure_one()
        ...
```
Just call the method on a single journal instead